committees/projects/index.md

```diff
@@ -6,8 +6,8 @@ The essence of Projects Committee is simple: some people want to work on things,
 and others want to do things, we make it happen.
 
 Got a project? Want to work on a project? Questions?  
-Contact projects at projects@sse.se.rit.edu
+Contact projects at <projects@sse.se.rit.edu>
```

Comment:
On a side note, this should be @sse.rit.edu because we're trying to fully remove the sse.se domain from our code.